TypeCobol/Compiler/Directives/CompilerDirective.cs

```diff
@@ -705,17 +705,22 @@ protected ReplaceOperation(ReplaceOperationType type, Token? comparisonToken)
     /// </summary>
     public class SingleTokenReplaceOperation : ReplaceOperation
     {
-        public SingleTokenReplaceOperation(Token? comparisonToken, Token? replacementToken)
+        public SingleTokenReplaceOperation(Token? comparisonToken, Token? replacementToken, Token? leadingToken = null, Token? trailingToken = null)
```

Comment:
In general avoid optional values. Here the constructor is called only once so we already have values provided.